dashboard/app/controllers/featured_projects_controller.rb

```diff
@@ -14,4 +15,40 @@ def unfeature
     @featured_project = FeaturedProject.find_by! storage_app_id: storage_app_id
     @featured_project.update! unfeatured_at: DateTime.now
   end
+
+  # Featured projects are selected internally for their
+  # quality, so we can be reasonably confident that they
+  # are not abusive. To prevent users from spamming Zendesk # with false reports of abuse on featured projects, this # sets their abuse score such that the project needs to
```

Comment:
nit: comment formatting --

```suggestion
  # are not abusive. To prevent users from spamming Zendesk 
  # with false reports of abuse on featured projects, this 
  # sets their abuse score such that the project needs to
```